src/config/dynamic-theme-fixes.config

```diff
@@ -3070,6 +3070,9 @@ CSS
 .generator_color.is-light {
     color: #000 !important;
 }
+.color-picker_saturation {
```

Comment:
I'm not able to find this element.

Reply:
It's in the palette generator part of the site, click generator and it should be in there.

To view the actual color picker, do the following:
Click the hex color number
Click the drop down arrow in that menu
Select Picker
![image](https://user-images.githubusercontent.com/18730519/142046818-327fc6fd-e42b-4df7-b142-332eed0806e8.png)